consul-topology/README.md

```diff
@@ -0,0 +1,197 @@
+[![GoDoc](https://pkg.go.dev/badge/github.com/hashicorp/consul-topology)](https://pkg.go.dev/github.com/hashicorp/consul-topology)
```

Comment:
Should we remove this badge?

Reply:
Or potentially fixup the URL to point at the new modules home?

Reply:
ah yep, I missed that when porting